cylc/flow/workflow_files.py

```diff
@@ -1326,6 +1326,11 @@ def validate_workflow_name(name: str) -> None:
             "Workflow name cannot be a path that points to the cylc-run "
             "directory or above"
         )
+    if re.findall('run(N|[0-9]+)', Path(name).name):
```

Comment:
Can't this be simplified to?
```suggestion
    if re.findall('run(N|\d+)', Path(name).name):
```

Reply:
Flake8 objects, unless you do 
```suggestion
     if re.findall(r'run(N|[\d+)', Path(name).name): 
```